style.css

```diff
@@ -721,3 +718,64 @@
   padding-left: 10px;
   margin-top: 20px;
 }
+
+.mobile {
+  background: #3c3a39;
+  color: #fff5e1;
+  font-size: 25px;
+  font-family: "Inter", sans-serif;
+  font-weight: 700;
+  padding: 20px;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: calc(100vw - 19px);
+  height: 100vh;
+}
+
+.peakMobile ul li {
+  margin-bottom: 25px;
+  padding-bottom: 10px;
+  border-bottom: 1px solid #6f6c6b;
+  list-style: none;
+}
+
+.close_btn img {
+  width: 20px;
+  height: 20px;
+}
+
+.close_btn {
+  display: flex;
+  justify-content: flex-end;
+  margin-bottom: 40px;
+}
+
+.close_btn:hover,
+.burger-icon:hover {
+  cursor: pointer;
+}
+
+.newMobile {
+  display: none;
+  position: absolute;
+  font-family: 'roboto', sans-serif;
+  font-weight: 700;
+  font-size: 45px;
+  top: 50px;
+  left: 0;
+  height: calc(75vh - 50px);
+  width: 100%;
+}
+
+.openMenu .newMobile {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  background: #3c3a39;
```

Comment:
- [ ] As already specified by the previous reviewer, please make sure the mobile menu contains the full height of the screen as specified in the original design. You can achieve this by giving it a `height` of `100vh`

### Your Design
![image](https://user-images.githubusercontent.com/43995830/157435176-d2a0f80a-29c0-4e00-9f95-6566278af6d3.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/157435105-8602bf89-0514-479f-9e7e-c57cba5e5e93.png)